app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -272,8 +277,193 @@ class NavigationDrawerActivityTest {
     }
   }
 
+  // TODO(#2535): Remove notation when Robolectric supports multiple times opening of the NavDrawer
+  // Requires opening the NavDrawer several times which is currently not supported on robolectric
+  @RunOn(TestPlatform.ESPRESSO)
+  @Test
+  fun testNavDrawer_openNavDrawer_clickSwitchProfile_clickCancel() {
```

Comment:
testNavDrawer_openNavDrawer_switchProfile_cancel_homeIsSelected

Reply:
Should I go with `testNavDrawer_openNavDrawer_switchProfile_cancel_homeItemIsSelected` or `testNavDrawer_openNavDrawer_switchProfile_cancel_homeItemIsSelected` ?

Reply:
Done

Reply:
Done